src/studio/src/designer/backend/Controllers/DatamodelsController.cs

```diff
@@ -222,14 +245,7 @@ public async Task<IActionResult> PutDatamodel(string org, string repository, [Fr
         {
             var developer = AuthenticationHelper.GetDeveloperUserName(HttpContext);
             var content = await ReadRequestBodyContentAsync();
-
-            if (saveOnly)
-            {
-                await _schemaModelService.UpdateSchema(org, repository, developer, modelPath, content, saveOnly);
-                return Ok();
-            }
-
-            await _schemaModelService.UpdateModelFilesFromJsonSchema(org, repository, developer, modelPath, content);
+            await _schemaModelService.UpdateSchema(org, repository, developer, modelPath, content, saveOnly);
```

Comment:
Please doublecheck if UpdateSchema is updated to use only new converters. Haven't been able to completely figure out in github diff.